packages/fx-core/tests/component/drivers/aad/create.test.ts

```diff
@@ -61,14 +61,14 @@ describe("aadAppCreate", async () => {
     args = {
       generateClientSecret: true,
     };
-    await expect(createAadAppDriver.run(args, mockedDriverContext))
+    await expect(createAadAppDriver.handler(args, mockedDriverContext))
```

Comment:
minor: in my opinion we should test the run() function instead of the handler function because run() is our interface. You can keep current code and I will update the test cases later.